src/shared/pager.c

```diff
@@ -53,6 +53,11 @@ noreturn static void pager_fallback(void) {
         _exit(EXIT_SUCCESS);
 }
 
+static int stored_stdout = -1;
+static int stored_stderr = -1;
+static bool stdout_redirected = false;
+static bool stderr_redirected = false;
```

Comment:
Those are not needed any more. `stdout_redirected` is equivalent to `stored_stdout >= 0`.